src/components/Card/Card.scss

```diff
@@ -34,13 +41,19 @@ $block: '.#{variables.$ns}card';
             pointer-events: none;
         }
 
-        &#{$block}_clickable:hover {
-            border-color: transparent;
-            cursor: pointer;
+        &#{$block}_clickable {
+            &:hover {
+                border-color: transparent;
+                cursor: pointer;
 
-            &::before {
-                content: '';
-                border: 2px solid var(--yc-color-line-selection-hover);
+                &::before {
+                    content: '';
+                    border: 2px solid var(--yc-color-line-selection-hover);
+                }
+            }
+
+            &:focus-within {
```

Comment:
Let's use the same technic used in the Button component, mouse click shouldn't trigger outline